### ISSUE TITLE
New category for solutions with 'hidden features'

### ISSUE THREAD

**Issue by commonnickname** [hingston/7-billion-humans-solutions#139]
There are some feature that don't exist in the game normally, but that you can introduce by editing the code in a text editor. For example: random pickUp, random set, numbers from 100 to 255 etc. 
However, putting solutions that use those features feels like it would defeat the purpose of having a repository where people can look up record solutions, - if those solutions cannot be written without 3rd party software. 
Or maybe not a new category, but a new tag: Year NUM - NAME (TYPE) - (in-game/editor).txt

**Comment by commonnickname** [comment]
Related questions: If we make a new category, what do you think we should do with solutions that use early random steps? Technically, a text editor is not required to write it, because it can be written on another level and then copied over, but I would be either way.

**Comment by n05ucc4u** [comment]
i think early random steps count as "hidden feature"

**Comment by commonnickname** [comment]
Another reason to have a new category is that in case the developers fix it, we won't have any problems retrieving older solutions that work.

**Comment by n05ucc4u** [comment]
yeah, that's right

**Comment by n05ucc4u** [comment]
but what about levels like compulsory office romance or levels with div 0 trick?

**Comment by commonnickname** [comment]
Or a category "without copying". Another motivation would be that the iPad version doesn't allow to copy and paste the code?

**Comment by NeunEinser** [comment]
I really think we should have this.
I personally do not like exploit solutions at all, and imo they do not belong in the main category, if a new category is created.
It might be the best solution, tho to have a new tag for exploit solutions, so that we do not have to create exploit categories for all 3 current categories.
For the overview table, a simple icon marker in the level name would probably work.

**Comment by Ormek** [comment]
I agree. I was looking for a speed solution for year _15:Shred Lines_. I found the top speed to be 11, but I cannot enter it in the game, just paste it, because the GUI does not allow me to create `pickup n,e`. Therefore, the answer was not fulfilling.

**Comment by Doomsday31415** [comment]
Why hasn't this been fixed yet?  It's been over a year.

Solutions that use exploits like pasting in invalid commands should be in a separate category, not the main one.

**Comment by blueglyph** [comment]
I agree, solutions that cannot be entered from the game, so either non-existent like `pickup n,e` or not granted at that level like `step n,e`, should be in a separate category, if there at all.

**Comment by abfipes12** [comment]
thanks for suggestion, I made a directory for pasteable-only solutions with 26 solutions

**Comment by ostcar** [comment]
The current best solution for Year 06 also uses a past-only commend. In this your, pickUp is only possible on the current tail, and not south.

https://github.com/hingston/7-billion-humans-solutions/blob/33623986795b304de516a0df9479e51baf835035/Solutions99%2B/Year%2006%20-%20Little%20Exterminator%201%20(both).txt#L11

**Comment by hingston** [comment]
Done, as a marker rather than a new folder. Solutions that cannot be typed into the game's editor are now marked 📋
in the tables, and CONTRIBUTING.md explains the rule and asks submitters to flag them.

A marker rather than a folder because it works across all three tables without needing a paste-only variant of each,
and because moving these solutions out would leave those levels with no entry at all until somebody submits a
typeable one — the information about what the best known solution is would be lost.

Seeded with the ones identified in this thread: Year 06 Little Exterminator 1 (`pickup c,s`), Year 15 Shred Lines
(speed) (`pickup n,ne`, the 11 step solution Ormek could not enter) and Year 19 Content Creators Bug Fix
(`takefrom sw,s`). Others almost certainly need the marker, but working out which command a level has unlocked needs
someone with the game in front of them — please open a pull request adding 📋 to any row you can confirm.

Changes are in #313.
